### DISCUSSION TITLE
Issue with Quotation Marks Handling in the read_csv() Function

### DISCUSSION THREAD

**Discussion by okankcb** [vertica/VerticaPy#1267] [Q&A] [answered]
Hello,

We encountered an unexpected behavior when using the read_csv() function with Verticapy 0.12 and 1.0.3.

**Problem Encountered:**
When importing a CSV file (attached to the discussion) with the following structure:
[produits.csv](https://github.com/user-attachments/files/16707021/produits.csv)

numero,libelle
1,Produit" A
2,Service B
3,"Pack ""C"" plus"
4,Offre D
5,"Solution ""E"" ""Premium"""

- When the libelle field contains a single double quote ("), it is interpreted as a separator, replacing the default comma.
- If a line contains an even number of double quotes, that line is not imported into the Vertica table, with no error or warning message.

**Temporary Solution:**
To work around this issue, we set the quotechar='' parameter. By default, this parameter is set as quotechar='"'.

However, this behavior does not seem normal and can be dangerous when importing data.

Here is an example of code that reproduces the issue:
```
read_csv("produits.csv", 
         table_name="TEST4",
         schema=schema,
         sep=",")
```
Out : 
![image](https://github.com/user-attachments/assets/8fa1716f-00f0-486b-822b-9a433ecc7267)

To resolve the issue, we modified the code as follows:
```
read_csv("produits.csv", 
         table_name="TEST4",
         schema=schema,
         sep=",",
         quotechar='')
```
Out :
![image](https://github.com/user-attachments/assets/fc0b484e-149e-4c62-b7d8-216b32360735)

It would be desirable to receive a "WARNING" to indicate a problem in these two cases, in order to avoid potentially incorrect results without the user's knowledge.

Would it be possible to examine on your end whether it is feasible to modify this behavior?

Thank you for your attention to this matter.

Best regards,
Okan.K

**Comment by oualib** [comment]
Thanks for reporting this issue, I'll do some tests and come back to you.

**Comment by oualib** [comment, marked as answer]
Hi Okan,

The behavior is expected, as the default string delimiter is ". In your case, changing this delimiter to an empty one should resolve the issue.

Since the parsing occurs at the server side, managing the warning without parsing the file twice will be challenging. I don’t consider this a bug.
